src/components/Card.tsx

```diff
@@ -65,13 +69,22 @@ const cardColor = (color?: Color) => {
   )
 }
 
+const appStyles = classnames(
+  space('space-y-6'),
+  display('flex'),
+  flexDirection('flex-col'),
+  alignItems('items-stretch'),
+  minHeight('sm:min-h-card', 'min-h-fit')
```

Comment:
Basically this applies the box to grow between `minHeight` and `maxHeight`

Reply:
+ we don't need to dance with `height` and `maxHeight`, only min-max. Max is defined without `forApp` prop